portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -132,12 +132,64 @@ definition {
 	}
 
 	@description = "Verify that a link to a URL can be added to the Body Text"
-	@ignore = "true"
 	@priority = "5"
 	test CanSupportALinkToAURL {
 		property portal.acceptance = "false";
```

Comment:
Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.

Reply:
Fixed